config/initializers/devise.rb

```diff
@@ -23,6 +23,7 @@
 
   # Configure the class responsible to send e-mails.
   # config.mailer = 'Devise::Mailer'
+  config.mailer = 'UserMailer'
 
   # Configure the parent class responsible to send e-mails.
   # config.parent_mailer = 'ActionMailer::Base'
```

Comment:
Here's the `parent_mailer` config.

Reply:
... I had changed the line directly above it, without noticing it. Embarrassing. :laughing: